index.html

```diff
@@ -1,60 +1,217 @@
+            project you need coded, don`t hestiate to contact me.
+            </p>
+          </div>
+          
+          <ul class="social-media">
+            <li><a href="#"><img src="./svg/github.svg" alt="github"/></a></li>
+            <li><a href="#"><img src="./svg/linkedin.svg" alt="linkedin"/></a></li>
+            <li><a href="#"><img src="./svg/Vector3.svg" alt="angelist"/></a></li>
+            <li><a href="#"><img src="./svg/twitter.svg" alt="twitter"/></a></li>
+            <li><a href="#"><img src="./svg/Vector5.svg" alt="medium"/></a></li>
+          </ul>
+        </div>
+        <div class="background-desktop">
+        </div>
+      </section>
+ 
+  </div>
+  <section class="work">
+    <a href="#work"></a>
 
-                    <ul class="social-media">
-                        <li><a href="#"><img src="./svg/github.svg" alt="github" /></a></li>
-                        <li><a href="#"><img src="./svg/linkedin.svg" alt="linkedin" /></a></li>
-                        <li><a href="#"><img src="./svg/Vector3.svg" alt="angelist" /></a></li>
-                        <li><a href="#"><img src="./svg/twitter.svg" alt="twitter" /></a></li>
-                        <li><a href="#"><img src="./svg/Vector5.svg" alt="medium" /></a></li>
-                    </ul>
-                </div>
-                <div class="background-desktop">
-                </div>
-            </section>
+    <div class="work-1">
+      <div class="work1-visual">
+        <h1>My Recent Works</h1>
+        <div class="line"></div>
+      </div>
+      <div class="work1-content">
+        <div class="work-placeholder"><img src="./svg/ImgPlaceholder.svg" alt="" />
         </div>
+        <div class="work-data">
+          <div class="content-data">
+            <h1>Multi-Post Stories</h1>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul>
+              <li class="list1"><a href="https://en.wikipedia.org/wiki/CSS">Css</a></li>
+              <li class="list1"><a href="https://en.wikipedia.org/wiki/HTML">Html</a></li>
+              <li class="list1"><a href="https://en.wikipedia.org/wiki/Bootstrap_(front-end_framework)">Bootstrap</a></li>
+              <li class="list1"><a href="https://en.wikipedia.org/wiki/Ruby_(programming_language)">Ruby</a></li>
+            </ul>
+            <div class="work1-btn"><button type="button" class="work1-button">See Project</button></div>
+          </div>
+        </div>
+      </div>
     </div>
-
+    <div class="work-display">
+    </div>
+    <div class="popUp-Container"></div>
+  </section>
+  <section class="about-me-section" id="about">
+    <div class="about-me-1">
+      <div class="about-me1-content">
+        <h1>About me</h1>
```

Comment:
- [ ] [REQUIRED] Please, make sure to use a `h2` or `h3` instead of `h1` for about me section because there should be only one `h1` in the whole page for semantic document